manager/notifier.go

```diff
@@ -570,6 +572,94 @@ func (n *notifier) sendEmailNotification(to string, op notificationOp, a *Alert)
 	return writeEmailBody(wc, *smtpSender, to, status, a, n.alertmanagerURL)
 }
 
+// opGenieMessageCreate is the request for sending an opsGenie notification. We are not specifying all the
+// fields. The Details field is populated with the labels from the alert.
+type opsGenieMessageCreate struct {
+	ApiKey      string                 `json:"apiKey"`
+	Message     string                 `json:"message"`
+	Description string                 `json:"description"`
+	Alias       string                 `json:"alias"`
+	Source      string                 `json:"source"`
+	Tags        []string               `json:"tags"`
+	Teams       []string               `json:"teams"`
+	Details     map[string]interface{} `json:"details"`
+}
+
+// opsGenieMessageClose closes an open alert in OpsGenie.
+type opsGenieMessageClose struct {
+	ApiKey string `json:"apiKey"`
+	Alias  string `json:alias"`
```

Comment:
[ Quoting <notifications@github.com> in "Re: [alertmanager] Alertmanager: Ad..." ]
>> +// opsGenieMessageClose closes an open alert in OpsGenie.
>> +type opsGenieMessageClose struct {
>> +	ApiKey string `json:"apiKey"`
>> +	Alias  string `json:alias"`
>
>(Oh, it's just the first quote that is missing - should be added)

Wow, sharp :) Yes, missing the quote. Fixed. And pushed again.

/Miek

--
Miek Gieben